docs/DORA.md

```diff
@@ -164,27 +150,42 @@ workflows:
       - deploy
 ```
 
-If you have set a [username/password](GettingStarted/Authentication.md) for Config UI, you need to add them to the curl to register a deployment:
 
-```
-curl https://sample-url.com/api/plugins/webhook/2/deployments -X 'POST' -u 'username:password' -d '{
-    \"commit_sha\":\"$CIRCLE_SHA1\",
-    \"repo_url\":\"$CIRCLE_REPOSITORY_URL\",
-    \"start_time\":\"$start_time\"
-  }'
-```
+2.5 Run the CircleCI pipeline. Visit DevLake's DB to check if the deployments have been successfully pushed to DevLake. The deployments will appear in table [cicd_deployments](DataModels/DevLakeDomainLayerSchema.md#cicd_deployments) and [cicd_deployment_commits](DataModels/DevLakeDomainLayerSchema.md#cicd_deployment_commits) in DevLake's database.
+
+  ![CircleCI-screenshot](https://i.imgur.com/IyneAMn.png)
+
+  ![webhook-query](/img/ConfigUI/webhook-query-deployments.png)
```

Comment:
Is it correct?  I cannot see the pic on preview mode.

<img width="602" alt="image" src="https://github.com/apache/incubator-devlake-website/assets/5844806/bb4702e3-8541-4469-8bc0-0d1403fd0313">
<img width="321" alt="image" src="https://github.com/apache/incubator-devlake-website/assets/5844806/dee4399a-c868-4d9d-9168-a3c732bd7d28">

Reply:
@d4x1  It should be a network problem. Never mind, I'll modify them to local files.